dietpi/dietpi-software

```diff
@@ -11249,13 +11249,15 @@ _EOF_
 			sed -i "/data.folder=/c\data.folder=$G_FP_DIETPI_USERDATA/blynk" $G_FP_DIETPI_USERDATA/blynk/server.properties
 
 			cat << _EOF_ > /etc/systemd/system/blynkserver.service
+
 [Unit]
 Description=Blynk Server
 After=network.target
 
 [Service]
 Type=simple
-ExecStart=$fp_java_binary -jar /etc/blynkserver/server.jar -serverConfig $G_FP_DIETPI_USERDATA/blynk/server.properties
+ExecStart=$fp_java_binary -jar /usr/local/bin/blynk/blynkserver.jar -serverConfig $G_FP_DIETPI_USERDATA/blynk/server.properties
```

Comment:
@joaofl 
I agree, similar to Syncthing, that a binary is better placed there, or in /opt/blynk or /mnt/dietpi_userdata/blynk to have everything in one place.
But similar to Syncthing, this is a larger task, since we carefully need to take care to properly migrate existing installs to the new location. This includes user information, so in case own scripts/guides and such need to be edited according to the new location.

But this is something we can add via separate PR then.

Reply:
Add everything to the `dietpi_userdata` folder to have everything in one place.